app/assets/stylesheets/base/_buttons.scss

```
@@ -0,0 +1,30 @@
+.button {
+  width: 100%;
+  padding: rem(10px) 0;
+  transition: $basic-transition;
+  border-radius: rem(55px);
+  text-align: center;
+
+  &--primary {
+    @include primary-gradient-background;
+
+    border: unset;
+    color: $white-500;
+    font-weight: 600;
+    text-transform: uppercase;
+
+    &:hover {
+      color: $black-100;
+    }
+  }
+
+  &--secondary {
+    border: rem(1px) solid $primary-dark;
```

```suggestion
    border: 1px solid $primary-dark;
```
It's a small number of pixels, we can omit that.
Because 1px when calculate with rem, we will get blur in UI 🫣

---

Gotchuu thanks for the advice 😄 Updated in https://github.com/aetheryna/google-scraper-ruby/pull/32/commits/c50b3c3e72925a76b1b328a88f4d9229fbb60f9b